jest.config.js

```diff
@@ -0,0 +1,7 @@
+const { defaults } = require('jest-config');
+
+module.exports = {
+  ...defaults,
+  setupFiles: ['./tests/globalMocks.js'],
+  testEnvironment: 'jsdom',
+};
```

Comment:
When I run your tests I get an error because the setup file `globalMocks.js` referenced in `jest.config.js` does not exist in your `tests` directory.

![image](https://user-images.githubusercontent.com/38088818/186947709-7eb3c9f1-af15-4023-9182-358ae7de7a14.png)

- Kindly look into it to fix this issue